### PR TITLE
Packages/ophidia server

### PR DESCRIPTION
**Pull request by SoniaScard**
Update Ophidia Server release to 1.7.4

### CODE REVIEW

**Review by eldoo — DISMISSED** [review]

File: var/spack/repos/builtin/packages/ophidia-server/package.py [hunk 1]
```diff
@@ -9,10 +9,10 @@
 class OphidiaServer(AutotoolsPackage):
     """Front-end server of the Ophidia framework"""
 
-    homepage = "https://github.com/SoniaScard/ophidia-server"
-    url = "https://github.com/SoniaScard/ophidia-server/archive/refs/tags/v1.7.2.tar.gz"
-    maintainers("eldoo", "SoniaScard")
-    version("1.7.2", sha256="452587775343b266bbb5adcfeee64e7f7e9a9bbfcb2133646a831ae3e74348be")
+    homepage = "https://github.com/OphidiaBigData/ophidia-server"
+    url = "https://github.com/OphidiaBigData/ophidia-server/archive/refs/tags/v1.7.4.tar.gz"
+    maintainers = ["eldoo", "SoniaScard"]
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/ophidia-server/package.py]: Why change this?  The previous (newer) syntax for `maintainers` is preferred.  

(https://spack.readthedocs.io/en/latest/packaging_guide.html#maintainers)

File: var/spack/repos/builtin/packages/ophidia-server/package.py [hunk 2]
```diff
@@ -9,10 +9,10 @@
...
+    version("1.7.4", sha256="30128c99ae089ab766141397ea5098ac930cfe10d09b289ed120f6581d8bb07d")
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/ophidia-server/package.py:15]: Normally versions are retained or at least deprecated before being removed (see https://spack.readthedocs.io/en/latest/packaging_guide.html#deprecating-old-versions).

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/ophidia-server/package.py:15]: You could restore `1.7.2` and add `, url="https://github.com/SoniaScard/ophidia-server/archive/refs/tags/v1.7.2.tar.gz"` along with `deprecated=True`.

**Review by tldahlgren — CHANGES_REQUESTED** [review]
Confirmed the new sha256.  Do have some issues with changing the `maintainers` syntax and a suggestion for deprecating the old version.

File: var/spack/repos/builtin/packages/ophidia-server/package.py (hunk 2 above)
Comment (tldahlgren) [reply var/spack/repos/builtin/packages/ophidia-server/package.py:15]: I think this package is new enough we could proceed without restoring `1.7.2`.  Just let me know what you want to do about this change.

Comment (SoniaScard) [reply var/spack/repos/builtin/packages/ophidia-server/package.py:15]: Yes, we could proceed without restoring 1.7.2 version.

**Review by tldahlgren — APPROVED** [review]

**Pull request merged**